client/gettext/json/available_languages.json

```diff
@@ -1 +1 @@
-{"es-ES":"Español","fr":"Français","ja":"日本語","ko":"한국어","nl":"Nederlands","pt-BR":"Português do Brasil  ","pt-PT":"Português","zh-Hans-CN":"简体中文","zh-TW":"國語"}
+{"es-ES":"Español (España)","es":"Español","fr":"Français","ja":"日本語","ko":"한국어","nl":"Nederlands","pt-BR":"Português do Brasil  ","pt-PT":"Português","zh-Hans-CN":"简体中文","zh-TW":"國語"}
```

Comment:
@mzazrivec travis still hates us, if string is indeed correct, the test ought to be updated https://github.com/ManageIQ/manageiq-ui-service/blob/ae385823137b78b78b1ea6b8abdee77b0b9b2ac4/client/app/core/language.service.spec.js#L78

Reply:
We have `es-ES` translation (not complete) and also `es` translation (100% complete). For product, we'll be enabling just `es`, I'm sure.

I wasn't aware of that test or what it does, though I can fix it to say ` expect(locale).to.eq('es')`

Reply:
yeah!! that should do it! 👍